app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -64,6 +64,7 @@ input CustomerAddressAttributeInput {
 
 type CustomerToken {
     token: String @doc(description: "The customer token")
+    expired_time: Int @doc(description: "Token lifetime (by hour)")
```

Comment:
I would suggest using `expiration_time` instead since `expired_time` might be literally translated as "time when the token has been expired" which does not reflect the actual value of this field 

```suggestion
    expiration_time: Float @doc(description: "Token lifetime (hours)")
```